bin/publish.sh

```diff
@@ -11,6 +11,8 @@ first_hub=$(echo $HUB | awk -F " " '{print $1}')
 TARGET_GOARCH=${TARGET_GOARCH:-${GOARCH:-amd64}}
 TARGET_GOOS=${TARGET_GOOS:-${GOOS:-linux}}
 CGO_ENABLED=${CGO_ENABLED:-0}
+BASE_IMAGE=${BASE_IMAGE-"ubuntu:bionic"}
```

Comment:
should we have this default value? I think it's better to let each module dockerfile have its own base img.

Reply:
in this case we can override this env from module's own publish.sh